### DISCUSSION TITLE
Q: 我用hexo搭建的博客想使用评论系统，但原装是waline v2，发送评论提示错误，怎么升级呢 || Q: I want to use the comment system for my blog built with hexo, but the original version is waline v2. When sending comments, it prompts an error. How can I upgrade?

### DISCUSSION THREAD

**Discussion by Peter267** [walinejs/waline#2651] [Q&A] [answered]
### Describe the bug | 问题描述

我用hexo搭建的博客想使用评论系统，但原装是waline v2，发送评论提示错误，怎么升级呢

### Website URL | 问题网站

https://peter267.zeabur.app

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I want to use the comment system for my blog built with hexo, but the original version is waline v2. When sending comments, it prompts an error. How can I upgrade?

### Website URL | Problem website

https://peter267.zeabur.app

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
serverURL 写错了，不是 Deployments 那个 url，是 Domain 对应的 url
